### PR TITLE
SonarQubue scanner updates for coverage

### PR DESCRIPTION
**Pull request by rstens**

### CODE REVIEW

File: backend/tfrs/settings.py [hunk 1]
```diff
@@ -102,6 +103,7 @@
 ]
 
 WSGI_APPLICATION = 'wsgi.application'
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
```
Comment (dainetrinidad) [inline backend/tfrs/settings.py]: @rstens Can we confirm this will still run the unit tests we have?

File: backend/Jenkinsfile-develop [hunk 2]
```diff
@@ -4,21 +4,15 @@ node('python') {
     stage('Unit Test') {
         checkout scm
         try {
-            sh 'cd backend && pip install --upgrade pip && pip install -r requirements.txt'
```
Comment (dainetrinidad) [inline backend/Jenkinsfile-develop]: @kuanfandevops can you confirm this works? I've noticed that `cd backend` was removed.

File: backend/tfrs/settings.py (hunk 1 above)
Comment (rstens) [reply backend/tfrs/settings.py]: Yes, this will run all the tests.

File: backend/Jenkinsfile-develop (hunk 2 above)
Comment (rstens) [reply backend/Jenkinsfile-develop]: You're right backend needs to be there!
I copied my code that does not do that.

**Comment by rstens** [comment]
Fixed the jenkinsfile to run the unit tests from the backend directory.

**Comment by kuanfandevops** [comment]
Peer reviewed. I'll merge the pr.

**Comment by kuanfandevops** [comment]
[Testing | Sonarqube Update for code coverage |2|](https://trello.com/c/DpzCgrRT/474-testing-sonarqube-update-for-code-coverage-2)

**Pull request merged**